src/Squirrel/ReleasePackage.cs

```diff
@@ -160,34 +160,42 @@ public string CreateReleasePackage(string outputFile, string packagesRootDir = n
         // We must do this, or PathTooLongException may be thrown for some unicode entry names.
         public static void ExtractZipDecoded(string zipFilePath, string outFolder, string directoryFilter = null)
         {
-            var zf = new ZipFile(zipFilePath);
+            Utility.Retry(() =>
```

Comment:
Another thought: do we know whether failure to extract/create directory will throw IOException or something else (a permission exception? a Zip utility exception?). May need to pass more exception types to Retry.